staging/kubefed/charts/controllermanager/templates/install-crds.yaml

```diff
@@ -0,0 +1,85 @@
+{{ if or (not .Values.global.scope) (eq .Values.global.scope "Cluster") }}
+{{if not (.Capabilities.APIVersions.Has "core.kubefed.io/v1beta1") }}
```

Comment:
We want this file to happen regardless of whether this capability exists otherwise we won't ever be able to upgrade the CRDs.